src/chain/member_info.rs

```diff
@@ -11,10 +11,15 @@
 pub struct AgeCounter(u32);
 
 impl AgeCounter {
+    #[cfg(any(test, feature = "mock_base"))]
     pub fn age(self) -> u8 {
         f64::from(self.0).log2() as u8
     }
 
+    pub fn set_age(&mut self, age: u8) {
+        self.0 = 2u32.pow(u32::from(age.max(MIN_AGE)))
```

Comment:
If we do need this cap, shall we make the function name more explicit?

Reply:
I'm not sure. Do you have a suggestion?

Btw, do you think it would be worth to add a `log_or_panic` when `age < MIN_AGE` there?

Reply:
maybe just a comment `This sets node's initial age which shall not be less than MIN_AGE` ?
Don't want to abuse `log_or_panic`, so current is fine   :)